netbox/utilities/forms/utils.py

```diff
@@ -40,7 +40,7 @@ def parse_numeric_range(string, base=10):
         except ValueError:
             raise forms.ValidationError(f'Range "{dash_range}" is invalid.')
         values.extend(range(begin, end))
-    return list(set(values))
+    return sorted(list(set(values)))
```

Comment:
I think sorted() should work directly on the set, without having to make a list.

```
>>> sorted(set([3,2,1]))
[1, 2, 3]
>>> sorted(list(set([3,2,1])))
[1, 2, 3]
```